podcasts/New Search/Model/SearchResultsModel.swift

```diff
@@ -14,6 +16,16 @@ class SearchResultsModel: ObservableObject {
 
     @Published var isShowingLocalResultsOnly = false
 
+    init(analyticsHelper: SearchAnalyticsHelper) {
+        self.analyticsHelper = analyticsHelper
+    }
+
+    #if DEBUG
```

Comment:
This is to avoid having to add the source to every preview. However, I changed it to be just a default constructor param in 9d4cc45b1cfab1f6c723f17e6ad4763d6f47f33a